runner/src/internal.rs

```diff
@@ -110,7 +110,7 @@ pub struct BuildClient {
 
 #[derive(Clone, Debug, PartialEq)]
 pub enum ServerVariant {
-    Base,
+    Unsafe,
     Coverage,
     Logless,
```

Comment:
I guess "logless" does not make much sense now, but there should be a "default" version included in this list?

Reply:
I agree. "default" is basically "Logless" now. I can rename "Logless" to "Default", and add an explanation somewhere in the docs. But it could get confusing since the default `server-variant` in the `runner` is (and I think should be) `unsafe`. Having "default" as a non-default option there could be confusing.  Perhaps we need a more expressive name instead of "default"?

Reply:
Maybe just call it base then? And we can still have the default being unsafe anyways.

Reply:
Yes. I think "Base" is a better name.